packages/office-ui-fabric-react/src/components/Check/Check.tsx

```diff
@@ -48,14 +48,14 @@ export class Check extends BaseComponent<ICheckProps, {}> {
           }
         ) }
       >
-        { Icon({
-          className: 'ms-Check-circle ' + styles.circle,
-          iconName: 'CircleRing'
-        }) }
-        { Icon({
-          className: 'ms-Check-check ' + styles.check,
-          iconName: 'StatusCircleCheckmark'
-        }) }
+        { <Icon
```

Comment:
The original intent here was that because Check is a hotspot, calling the stateless component directly was a performance optimization. I think the perf optimization is dated though, so I think this is fine.

I made a codepen to test the theory about if JSX or function calls are faster. They look pretty comparable.

https://codepen.io/dzearing/pen/xYpjrQ

Reply:
That's for creation, is there also a management overhead later when removing etc?

Reply:
The example also has a "clear" button which also measures, and I didn't see a noticeable impact.